__react/utils/bindings.js

```diff
@@ -0,0 +1,79 @@
+export function includeUnsubAfterForSubscription(message) {
```

Comment:
It would be great to have some docs explaining what this function is doing.

Reply:
You are right, I'll add jsdocs to all the functions.